Rendering/VolumeOpenGL2/vtkVolumeShaderComposer.h

```diff
@@ -1838,6 +1844,15 @@ inline std::string ComputeColorDeclaration(vtkRenderer* vtkNotUsed(ren),
           \n  }");
     return shaderStr;
   }
+  else if (noOfComponents == 4 && !independentComponents)
+  {
+    shaderStr += std::string("\
+          \nvec4 computeColor(vec4 scalar, float opacity)\
+          \n  {\
+          \n  return clamp(computeLighting(vec4(scalar.xyz, opacity), 3, 0.0), 0.0, 1.0);\
```

Comment:
Can we consider that using alpha for shading is what makes more sense for every use cases? Then this part should be contributed back to VTK ideally.

Reply:
I think yes, if there's an alpha channel you almost always want to use that for lighting, shading, and opacity calculations.  I don't know of any use case where you wouldn't want that.  And yes, this should go in VTK once we finalize it.